src/plots.py

```diff
@@ -325,3 +325,30 @@ def plot_donut(df, col_color):
             },
         },
     )
+
+
+def plot_line(df, col_x, col_y, col_color):
+    st.vega_lite_chart(
+        data=df,
+        spec={
+            **CONFIG_MAIN,
+            "mark": {
+                "type": "line",
+                "point": "true",
+                "tooltip": {"content": "data"},
+            },
+            "encoding": {
+                "x": {
+                    "field": col_x,
+                    "type": "quantitative",
+                },
+                "y": {
+                    "field": col_y,
+                    "type": "quantitative",
+                },
+                "color": {
+                    "field": col_color,
+                },
```

Comment:
I would suggest the following modifications to have even nicer plots (binning the x-axis and showing the average on y-axis)

```suggestion
            "mark": {
                "type": "line",
                "point": "true",
                "tooltip": True,
            },
            "encoding": {
                "x": {"field": col_x, "type": "quantitative", "bin": True},
                "y": {"field": col_y, "type": "quantitative", "aggregate": "mean"},
                "color": {"field": col_color, "type": "nominal"},
```

Reply:
Then, I think we are good to merge to main, good job! :rocket:

Reply:
> > any updates?
> 
> Could you update the code with the suggested changes above?

There was a problem with my code review, it should now appear correctly